CMakeLists.txt

```diff
@@ -11,8 +11,8 @@ option(DEBUG "Compile with debugging information." OFF)
 option(PROFILE "Compile with profiling information." OFF)
 option(ARMA_EXTRA_DEBUG "Compile with extra Armadillo debugging symbols." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
-option(BUILD_TESTS "Build tests. (Note: time consuming!)" OFF)
-option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
+option(BUILD_TESTS "Build tests. (Note: time consuming!)" ON)
+option(BUILD_CLI_EXECUTABLES "Build command-line executables." OFF)
```

Comment:
Oops, this probably shouldn't be committed (at least as part of this PR, if you actually wanted to make these changes I wouldn't mind discussing, but elsewhere).